### PR TITLE
Wizard: Update Review step

### LINKED ISSUE #774: Wizard: Review step updates
Update the Review step to match the updated mocks: https://marvelapp.com/prototype/6g7dja9/screen/82042200

### PR DESCRIPTION
**Pull request by regexowl**
Fixes #774. This matches the Review step with the updated mocks as per SPUR.

The changes are following:
1. Target environment tab:
	- value of AWS `Account ID` aligned with GCP and Azure values
Screenshot from 2022-11-07 15-57-54

2. Registration tab:
	- `Subscription` changed to `Registration type`
	- `Organization ID` removed, will be added in a different place so it's clearer it's associated with an activation key
Screenshot from 2022-11-07 15-58-02

3. System configuration:
	- updated column name `Type` to `File system type` in partitions popover
	- updated the width of the column to accommodate the change
	- unified line spacing of `Image size` with the other rows
	- changes `Packages` to `Additional packages`
Screenshot from 2022-11-07 15-58-10
Screenshot from 2022-11-07 15-58-16

### CODE REVIEW

**Comment by regexowl** [comment]
Additional information for an activation key is in a separate issue #822 as it should be used both in Registration and Review step.

File: src/Components/CreateImageWizard/CreateImageWizard.scss [hunk 1]
```diff
@@ -48,3 +48,7 @@
 .pf-c-select__menu-item.pf-m-load {
     --pf-c-select__menu-item--Color: var(--pf-global--Color--100);
 }
+
+.pf-u-min-width {
+    --pf-u-min-width--MinWidth: 11ch;
+}
```
Comment (croissanne) [inline src/Components/CreateImageWizard/CreateImageWizard.scss:54]: Nitpick: 1 trailing newline

**Review by croissanne — CHANGES_REQUESTED** [review]
LGTM overall, but could you edit the commit message to detail the exact changes (just a little bullet-point list or something), and maybe include a screenshot in the PR description?

File: src/Components/CreateImageWizard/CreateImageWizard.scss (hunk 1 above)
Comment (regexowl) [reply src/Components/CreateImageWizard/CreateImageWizard.scss]: Fixed!

**Review by croissanne — APPROVED** [review]
perfect, thank you!

**Pull request merged**